pdns/auth-main.cc

```diff
@@ -96,9 +96,9 @@ string g_programname = "pdns"; // used in packethandler.cc
 const char* funnytext = "*****************************************************************************\n"
```

Comment:
I don't suppose you could make this change:
```suggestion
const char* funnyText = "*****************************************************************************\n"
```

Or `funny_text`